examples/vision/metric_learning.py

```diff
@@ -192,14 +192,17 @@ def train_step(self, data):
             # pairs, to be high. This loss will move embeddings for the
             # anchor/positive pairs together and move all other pairs apart.
             sparse_labels = keras.ops.arange(num_classes)
-            loss = self.compiled_loss(sparse_labels, similarities)
+            loss = self.compute_loss(x=None, y=sparse_labels, y_pred=similarities)
```

Comment:
Tip: you can omit `x=None` here. Not important though

Reply:
Removed!